API/Services/Tasks/VersionUpdaterService.cs

```diff
@@ -90,12 +90,14 @@ public async Task<IEnumerable<UpdateNotificationDto>> GetAllReleases()
         private UpdateNotificationDto CreateDto(GithubReleaseMetadata update)
         {
             if (update == null || string.IsNullOrEmpty(update.Tag_Name)) return null;
-            var version = update.Tag_Name.Replace("v", string.Empty);
-            var updateVersion = new Version(version);
+            var updateVersion = new Version(update.Tag_Name.Replace("v", string.Empty));
+            var currentVersion = updateVersion.Revision == -1
+                ? BuildInfo.Version.ToString().Substring(0, 5)
```

Comment:
Don't substring as if we have 0.10.1.1, it will break. If anything, you can substring(0, BuildInfo.Version.ToString().LastIndexOf("."))

but move the BuildInfo.Version.ToString() to a local variable.

Reply:
Duh, don't know how I didn't think of that. Made the change.